Makefile

```diff
@@ -89,16 +92,22 @@ endif
 include contrib/devtools/Makefile
 
 ###############################################################################
-###                              Documentation                              ###
+###                              Build                                      ###
 ###############################################################################
 
+check_version:
+ifneq ($(GO_MINOR_VERSION),18)
```

Comment:
Would there be a problem with checking against `1.18` or even `>= 1.18`?

Also, would it be possible to make the hardcoded `18` const into a var?
```bash
REQUIRE_GO_VERSION = '1.18'
```